inspec/a2-api-integration/controls/iam_v2.rb

```diff
@@ -1094,6 +1088,33 @@
         expect(resp.parsed_response_body[:rule]).to eq(CUSTOM_RULE_1)
       end
 
+      it "GET /iam/v2beta/projects/:id/rules returns the most up-to-date rules for the project" do
```

Comment:
here, too, "up-to-date" is probably only meaningful to us now bc we're in the thick of it. I'd be more specific about staged vs applied (or something)